src/utils/helpers.ts

```diff
@@ -1,3 +1,5 @@
 // Add helpers here. This is usually code that is just JS and not React code. Example: write a function that
 // calculates number of minutes when passed in seconds. Things of this nature that you don't want to copy/paste
 // everywhere.
+
+//HELPERS WERE FEW LINES OF CODE SO LEFT THEM INTEXT.
```

Comment:
doesn't matter if its a few lines, start early to avoid having to do it afterwards